internal/validation/apiextensions/v1/composition/handler_test.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by client-gen. DO NOT EDIT.
+package composition
 
-// This package has the automatically generated clientset.
-package versioned
+import "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+var _ admission.CustomValidator = &validator{}
```

Comment:
Why the need to have this in a test file instead of directly in internal/validation/apiextensions/v1/composition/handler.go?

Reply:
Because it's a test. 🙂 We're testing that our `*validator` type satisfies the interface we need it to satisfy. No need for this code to be part of our production binary.